app/services/weets.js

```diff
@@ -1,6 +1,14 @@
 const { default: Axios } = require('axios');
+const logger = require('../logger');
+
+const config = require('../../config');
 
 exports.getQuote = async () => {
-  const { data } = await Axios.get('https://geek-jokes.sameerkumar.website/api?format=json');
-  return data;
+  try {
+    const { data } = await Axios.get(config.common.api.randomWeetsApi);
+    return data;
+  } catch (error) {
+    logger.error(error);
+    return error;
```

Comment:
Do not return the error directly, create a custom error, I had told you that for that you could look at the `error.js` that is at the root of the app folder.